src/trios/TRIOS_Domain.C

```diff
@@ -509,6 +518,72 @@ namespace TRIOS
 #endif //}
     }
 
+    double Domain::getXpos(double x) const
+    {
+        double dx = (xmax-xmin)/n;
+        return x*dx + xmin;
+    }
+
+    double Domain::getYpos(double y) const
+    {
+        double dy = (ymax-ymin)/m;
+        return y*dy + ymin;
+    }
+
+    double Domain::getRawZpos(double z) const
```

Comment:
It is not clear why these have "Raw" in the name and the others do not.

Reply:
This is just the Z position scaled with `dz` which is necessary for a bunch of derived values. The actual *physical* Z position has to go through the stretching function, so there will be a `getZpos` which (is based on this raw Z position), but I didn't implement/test that yet.

Reply:
I'd think this one should be protected then?